app/src/main/java/com/github/polypoly/app/ui/menu/SignInActivity.kt

```diff
@@ -178,7 +175,7 @@ class SignInActivity : ComponentActivity() {
      */
     @Composable
     private fun SignInButton() {
```

Comment:
Since the guest is not linked to any user, if the guest wants to play he must create a quick profile so that he at least has a name. For the moment, this only allows to say "welcome" with the name he wants, but later on, we could imagine that the guest can also play the games. It was just too big a task for me for the sprint if I implemented all of this at once, so I just did the minimum viable product, even if it doesn't look like a dream.

Of course, create profile can be implemented when we can register in addition to sign in. But this was simply not the object of my task. I simply modularized CreateProfileActivity enough so that it can be used in both cases.

Reply:
Sounds fair to me, thanks for the clear explanation :)